.github/workflows/flex_deploy_dev.yaml

```diff
@@ -9,28 +9,26 @@ on:
   # Enable running this workflow manually from the Actions tab
   workflow_dispatch:
 
-env:
-  ENVIRONMENT: dev
-  TWILIO_ACCOUNT_SID: ${{ secrets.TWILIO_ACCOUNT_SID_DEV }}
-  TWILIO_API_KEY: ${{ secrets.TWILIO_API_KEY_DEV }}
-  TWILIO_API_SECRET: ${{ secrets.TWILIO_API_SECRET_DEV }}
-  PLUGIN_FOLDER: ${{ secrets.PLUGIN_FOLDER }}
-  OVERWRITE_CONFIG: ${{ secrets.OVERWRITE_CONFIG }}
 
 jobs:
   deploy-serverless:
     runs-on: ubuntu-latest
+    environment: dev
+    env:
+      ENVIRONMENT: dev
```

Comment:
Now that we are using GitHub environments, we can eliminate the env-specific workflows and instead use a workflow input to define the environment. That doesn't need to happen in this PR though.